ktor-client/ktor-client-features/ktor-client-auth/common/src/io/ktor/client/features/auth/providers/BearerAuthProvider.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2014-2020 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.features.auth.providers
+
+import io.ktor.client.features.auth.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.http.auth.*
+
+/**
+ * Add [BasicAuthProvider] to client [Auth] providers.
+ */
+fun Auth.bearer(block: BearerAuthConfig.() -> Unit) {
+    with(BearerAuthConfig().apply(block)) {
+        providers.add(BearerAuthProvider(refreshTokenFun, loadTokensFun, true, realm))
+    }
+}
+
+data class Tokens(
+    val accessToken: String,
+    val refreshToken: String,
+    val type: String
```

Comment:
Why do we need type here?
If it's always "Bearer" maybe it worth renaming the class to `BearerTokens` and removing the `type` param?

Reply:
Agree. Will fix it